crates/duty-tracker/src/contract_manager.rs

```diff
@@ -256,7 +258,7 @@ impl ContractManager {
                     }
                     Err(e) => {
                         error!(%blockhash, %cursor, %e, "failed to process block");
-                        break;
+                        panic!("{:?}", e);
```

Comment:
I think you can do something like:
```suggestion
                        panic!("{e:?}");
```